traitcuration/static/js/listjs/list.js

```diff
@@ -0,0 +1,2 @@
+// List.js v1.4.1 (http://www.listjs.com) by Jonny Strömberg (http://javve.com)
```

Comment:
The same comment applies to this library, depending on which decision we will go with in #35: either use external CDN, or install via Yarn, or keep locally and include update/installation instructions